example/lib/main.dart

```diff
@@ -54,7 +54,7 @@ class MapsDemo extends StatelessWidget {
   //
   // Alternatively you can replace `String.fromEnvironment("ACCESS_TOKEN")`
   // in the following line with your access token directly.
-  static const String ACCESS_TOKEN = "pk.eyJ1IjoibWFpb3MiLCJhIjoiY2wwczRydDdvMDAwYzNkbnZ6MmNoOXM4aCJ9.W62RkvsUQSnqX86JLFzo1w";
```

Comment:
I think you'll have to drop this commit and the original one, as reverting still leaves this token in git history.

Reply:
good point, done ✅